src/platform/elements/select/Select.ts

```diff
@@ -180,65 +185,65 @@ export class NovoSelectElement implements OnInit, OnChanges {
+      if (!this.panelOpen) {
+        this.openPanel();
       }
-      if (event.keyCode === KeyCodes.ENTER) {
-        if (this.header.open && this.header.value) {
-          this.saveHeader();
-          return;
-        }
-        this.setValueAndClose({ value: this.filteredOptions[this.selectedIndex], index: this.selectedIndex });
-        return;
+      this.selectedIndex--;
+      this.select(this.filteredOptions[this.selectedIndex], this.selectedIndex);
+      this.scrollToSelected();
+    } else if (event.keyCode === KeyCodes.DOWN && this.selectedIndex < this.filteredOptions.length - 1) {
+      if (!this.panelOpen) {
+        this.openPanel();
       }
-
-      if (event.keyCode === KeyCodes.UP && this.selectedIndex > 0) {
-        this.selectedIndex--;
-        this.select(this.filteredOptions[this.selectedIndex], this.selectedIndex);
-        this.scrollToSelected();
-      } else if (event.keyCode === KeyCodes.DOWN && this.selectedIndex < this.filteredOptions.length - 1) {
-        this.selectedIndex++;
-        this.select(this.filteredOptions[this.selectedIndex], this.selectedIndex);
-        this.scrollToSelected();
-        if (this.header.open) {
-          this.toggleHeader(null, false);
-        }
-      } else if (event.keyCode === KeyCodes.UP && this.selectedIndex === 0) {
-        this.selectedIndex--;
-        this.toggleHeader(null, true);
-      } else if ((event.keyCode >= 65 && event.keyCode <= 90) || event.keyCode === KeyCodes.SPACE) {
-        clearTimeout(this.filterTermTimeout);
-        this.filterTermTimeout = setTimeout(() => {
-          this.filterTerm = '';
-        }, 2000);
-        let char = String.fromCharCode(event.keyCode);
-        this.filterTerm = this.filterTerm.concat(char);
-        // let element = this.element.nativeElement;
-        // let list = element.querySelector('.novo-select-list');
-        // let item = element.querySelector(`[data-automation-value^="${this.filterTerm}" i]`);
-        let item = this.filteredOptions.find((i) => i.label.toUpperCase().indexOf(this.filterTerm) === 0);
-        if (item) {
-          this.select(item, this.filteredOptions.indexOf(item));
-          this.scrollToSelected();
-        }
-      } else if ([KeyCodes.BACKSPACE, KeyCodes.DELETE].includes(event.keyCode)) {
-        clearTimeout(this.filterTermTimeout);
-        this.filterTermTimeout = setTimeout(() => {
-          this.filterTerm = '';
-        }, 2000);
-        this.filterTerm = this.filterTerm.slice(0, -1);
+      this.selectedIndex++;
+      this.select(this.filteredOptions[this.selectedIndex], this.selectedIndex);
+      this.scrollToSelected();
+      if (this.header.open) {
+        this.toggleHeader(null, false);
       }
-    } else {
-      if ([KeyCodes.DOWN, KeyCodes.UP].includes(event.keyCode)) {
-        this.panelOpen ? this.closePanel() : this.openPanel();
+    } else if (event.keyCode === KeyCodes.UP && this.selectedIndex === 0) {
```

Comment:
You will probably want to swallow the events for down/up too, otherwise when they get to the top of the list the window will scroll.